sw/device/silicon_creator/rom/e2e/rom_e2e_bootstrap_rma_test.c

```diff
@@ -0,0 +1,73 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/rstmgr_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+static const char *reset_reason_to_string(dif_rstmgr_reset_info_t reset_info) {
```

Comment:
nit: I think this utility probably belongs to a testutils file (sw/device/lib/testing/rstmgr_testutils.{h,c}), WDYT?

Reply:
N/a since I've deleted this code now.